dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityService.java

```diff
@@ -50,6 +50,9 @@ FileResourceStream getFileResourceImage(
       UID trackedEntity, UID attribute, UID program, ImageFileDimension dimension)
       throws NotFoundException;
 
+  TrackedEntity getTrackedEntity(String uid)
```

Comment:
Should we add a javadoc to all our exporters that the operations are done using the current user. I added something like that when adding the user in the param to explain the difference between the ones with a user and the ones without. I think it would still be useful to tell clients that this is what our API does.

Reply:
Done